src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -42,9 +41,34 @@ public function getAccount() {
    * Set the current object account.
    *
    * @param AccountInterface $account
+   *   The user object.
+   * @return $this
    */
-  public function setAccount(AccountInterface $account) {
+  public function setAccount( $account) {
     $this->currentUser = $account;
+    return $this;
+  }
+
+  /**
+   * Get the field configuration.
+   *
+   * @param $key
+   *   Specific key from the configuration. Optional.
+   *
+   * @return array|string
+   *   Value of a configuration or all the configurations.
+   */
+  public function getConfiguration($key = NULL) {
+    return empty($this->configuration[$key]) ? $this->configuration : $this->configuration[$key];
+  }
+
+  /**
+   * Get the handler of the field.
+   *
+   * @return DefaultSelection
+   */
+  public function getHandler() {
+    return \Drupal::service('plugin.manager.entity_reference_selection')->getInstance($this->configuration);
```

Comment:
But i should call our own plugin manager - https://github.com/amitaibu/og/pull/11/files#diff-74d260cea47b42dffdde5e684823d336R86